pkg/scheduler/framework/framework.go

```diff
@@ -45,16 +46,19 @@ func OpenSession(cache cache.Cache, tiers []conf.Tier, configurations []conf.Con
 			}
 		}
 	}
+	metrics.UpdatePluginDuration("total", metrics.OnSessionOpen, metrics.Duration(sessionStartTs))
```

Comment:
Can you give some interpretations about these metrics? Personally speaking, I don't think we should add a metric for process of `openSession`.

Reply:
The main goal is to make it easier to analyze key performance issues. For example, if there are 10 plugins and we have metrics of open and close for each of them, then there are 20 curves. When we want to know if open/close is the performance bottleneck, we want to see the overall latency of open and close first instead of each plugin; furthermore, if we have determined that open/close is the performance bottleneck, we will look at the performance of each plugin, at which time we will see 10/20 curves. If there is a curve of total usage at this time, it will be easier to visually perceive the performance of each plugin.